### ISSUE TITLE
Where are the Java communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/java#1467]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Java supported by one or more large organizations?
* Does Java have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Java? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Java? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Java? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Java?
* Are there popular newsletters for Java?
* Is Java taught at programming bootcamps? (If so, what are some examples?)
* Is Java taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Java?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by FridaTveit** [comment]
> Is Java taught at universities? (If so, what are some examples?)

Java is one of the main languages taught in the Computer Science course at Cambridge.

**Comment by sjwarner-bp** [comment]
It was taught pretty heavily on my course at Warwick too! Additionally, from our [ABOUT.md](https://github.com/exercism/java/blob/master/docs/ABOUT.md)...

> Java is among the most popular available programming languages, thanks to its versatility and compatibility. It is widely used for software development, mobile applications and developing larger systems.
> 
> Java was born in 1995 and is maintained by Oracle. Despite the fact that it isn't as young as some of the fresh languages out there, Java is still really popular. It was designed to be fast, secure, reliable, beginner-friendly and highly portable. This portability perk exists because Java is executed on a cross-platform compatible Java Virtual Machine - JVM. Android apps are also developed using Java, since the Android Operating System runs on a Java language environment.
> 
> The Java community is huge! GitHub for example has over 1.5 million Java projects. It's also worth mentioning that Java has the second largest community in StackOverflow! This is important because the larger a programming language community is, the more support you'd be likely to get.
> 
> Java also has a powerful and well-designed set of built-in APIs - Application Programming Interfaces, which can be used for various activities like Database connection, networking, I/O, XML parsing, utilities, and much more.
> 
> From laptops to datacenters, game consoles to scientific supercomputers, cell phones to the Internet, Java is everywhere!

It might even be worth updating this file if anything new jumps out over here :slightly_smiling_face: 

I've been to a couple of good conferences for Java ([QCon London](https://www.qconlondon.com), probably all of QCon, (Devoxx)[https://www.devoxx.co.uk/]), and the whole Java Champion thing is probably worth mentioning because those people are a great place to learn.

[InfoQ](https://www.infoq.com/) offer newsletters which are often somewhat Java-centric and normally a good read.

If I remember correctly, Devoxx also offered a day for kids to come along, try things out and learn about programming (sounded like a great idea)! Additionally, the [JDuchess](http://www.jduchess.org/) group exist for women in Java technology.

As for a community manager, I'm not too sure if one exists. Oracle do however have a community section for Java on their website (here)[http://www.oracle.com/technetwork/java/community/index.html].

If anything else comes to me I'll try and update it. If we want to formalise this and document it, I can tidy up my messy thoughts and ramblings too :slightly_smiling_face:

**Comment by kytrinyx** [comment]
Awesome thank you!

**Comment by kytrinyx** [comment]
(Oh, and https://javaranch.com/ has HUGE forums that have been around since the dawn of time, or thereabouts, with lots of friendly, helpful moderators).
